src/App.js

```diff
@@ -1,25 +1,46 @@
-import logo from './logo.svg';
 import './App.css';
+import Form from "./Components/Form";
+import Navbar from "./Components/Navbar";
+import {useEffect, useState} from "react";
+import Journeys from "./Components/Journeys";
```

Comment:
It's quite nice if you group all your components together and list them alphabetically.